3_javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,131 @@
+    },
+    {
+    name:"Julia",
+    width:5,
+    depth:11,
+    height:8,
+    gardenSizeInM2:70,
+    houseCost:1000000,
+    }    
+];
+
+function calculateHousePrice(house) {
+    return house.width*house.depth*house.height*2.5*1000 + house.gardenSizeInM2*300;
+}
+function dealOrNoDeal(house) {
+    const housePrice = calculateHousePrice(house);
+    if (housePrice>house.houseCost) {
+        console.log(`${house.name} is being ripped off by the seller`);
+    }
+    else
+    {
+        console.log(`${house.name} is getting a life changing deal`);
+    }
+}
+houseDetails.forEach(dealOrNoDeal);
+
+//     const peterHousePrice = 
+//     houseDetails[0].width*
+//     houseDetails[0].depth*
+//     houseDetails[0].height*
+//     2.5*1000 + houseDetails[0].gardenSizeInM2*300;
+
+//     const juliaHousePrice = 
+//     houseDetails[1].width*
+//     houseDetails[1].depth*
+//     houseDetails[1].height*
+//     2.5*1000 + houseDetails[1].gardenSizeInM2*300;
+    
+// if (peterHousePrice > houseDetails[0].houseCost) {
+//     console.log(`${houseDetails[0].name} is paying too much for his house`
+//     );
+// }
+// else 
+// {
+//     console.log(`${houseDetails[0].name} is getting a good deal for his money`
+//     );
+// }
+
+// if (juliaHousePrice > houseDetails[1].houseCost) {
+//     console.log(`${houseDetails[1].name} is paying too much for her house`
+//     );
+// }
+// else 
+// {
+//     console.log(`${houseDetails[1].name} is getting a good deal for her money`
+//     );
+// }
+
+// Ez Namey (Startup name generator) Optional
+// At a meetup you overhear a conversation between two developers. It went something like this: "Man i wish picking a startup name was easier! You need to be creative, funny and interesting, its nearly impossible!" Another problem to solve, awesome!
+// Lets help people who struggle finding a startup name by creating some code!
+// Create two arrays called firstWords, secondWords. The arrays should have 10 elements containing strings of possible startup names. Some examples could be: "Easy", "Awesome", "Corporate".
+// Create a variable called startupName that will contain the created startup name.
+// Using a random index (you choose) of the arrays and concatenation of strings, create and log the new startup name and the number of characters in it. An example could be: "The startup: "Easy Corporation" contains 16 characters"
+// Hint: you can use this code to generate a random number from 0-9, if you dont want to specify the indexes yourself.
+// const randomNumber = Math.floor(Math.random() * 10);
+
+let firstWords = [
+    "Easy",
+    "Awesom",
+    "Corporate",
+    "Bint",
+    "Schizoid",
+    "Scottish",
+    "Old",
+    "Woman",
+    "Arabian",
+    "Knights",
+
+
+];
+let lastWords = [
+    "Bimbo",
+    "Himbo",
+    "Widow",
+    "Tales",
+    "Rider",
+    "Dudes",
+    "Awakening",
+    "Laassi",
+    "Gringa",
+    "Dandy",
+];
+const startupName = firstWords[4] + " " + lastWords[0];
```

Comment:
you could try to use Math.floor(Math.random() * 10 here :) to show different result